staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -131,8 +131,7 @@ func ObjectReaction(tracker ObjectTracker) ReactionFunc {
 		case PatchActionImpl:
 			obj, err := tracker.Get(gvr, ns, action.GetName())
 			if err != nil {
-				// object is not registered
-				return false, nil, err
+				return true, nil, err
```

Comment:
LGTM. The previous code seemed to be an oversight. 

@caesarxuchao Does the boolean exist only to allow us skipping some valid actions in [chain](https://github.com/kubernetes/kubernetes/blob/3c5c6025bf5e962dd67931fd38e18031108d5a37/staging/src/k8s.io/client-go/testing/fake.go#L141-L143) which don't have reaction defined (Watch, ProxyGet)?

Reply:
If you look at the rest of the function you’ll see that it’s only used for unknown actions, so yes it seems to be an oversight.

Reply:
surprise! 🙃unhandling patch events here makes the fake client working correctly w/ the cohibitating resources! the fake client is simply grouping requesting objects by its `gvr` so the test is actually suppose to be broken when we write w/ `extensions/v1` but read w/ `apps/v1`.